doc/api_assets/api.js

```diff
@@ -136,6 +136,30 @@
     updateHashes();
   }
 
+  function setupFlavorSelectors() {
+    const kFlavorPreference = 'customFlavor';
+    const vCommonJS = 'cjs';
+    const flavorSetting = sessionStorage.getItem(kFlavorPreference);
+    const flavorSelectors = document.querySelectorAll('.js-flavor-selector');
+
+    flavorSelectors.forEach((selector) => {
+      selector.checked = flavorSetting !== vCommonJS;
+      selector.addEventListener('click', (e) => {
```

Comment:
I find it a bit confusing to call them "selector", because selector usually refers to a CSS selector.
```suggestion
    const flavorToggles = document.querySelectorAll('.js-flavor-selector');

    flavorToggles.forEach((toggleElement) => {
      toggleElement.checked = flavorSetting !== vCommonJS;
      toggleElement.addEventListener('click', (e) => {
```

Reply:
Elsewhere in the file it's referred to as "selector", see the "flavorSelector" function in the same file. Would maybe be even more confusing if it's not the same as in the other function. The class name also ends with "selector".

I tried to mimic the style of the other functions to increase coherency.

Reply:
But commit your suggestion if you feel like it's better, it's fine by me.

Reply:
I opened https://github.com/nodejs/node/pull/49536, let's see what the others think